src/platform/utilities/api/index.js

```diff
@@ -1,8 +1,62 @@
 import * as Sentry from '@sentry/browser';
 
 import environment from '../environment';
+import ENVIRONMENTS from '../../../site/constants/environments';
 import localStorage from '../storage/localStorage';
 
+const SSOeEndpoint = () => {
+  let environmentPrefix;
+  switch (ENVIRONMENTS.BUILDTYPE) {
+    case ENVIRONMENTS.VAGOVSTAGING:
+      environmentPrefix = 'sqa.';
+      break;
+    case ENVIRONMENTS.VAGOVDEV:
+      environmentPrefix = 'int.';
+      break;
+    default:
+      environmentPrefix = '';
+  }
+
+  return `https://${environmentPrefix}eauth.va.gov/keepalive`;
+};
+
+function refreshSSOeSession(sessionExpiration) {
+  const minutesUntilExpiration =
+    (sessionExpiration.getTime() - Date.now()) / (1000 * 60);
+
+  if (minutesUntilExpiration > 10) {
```

Comment:
@hinzed1127 not a huge concern, but if they IAM team ever changes the default timeout (eg 15 mins -> 10 mins), this method would never get triggered and we'd end up pinging the keepalive enpoint on every user request.  An alternative approach would be set a threshold based on a percentage of the `session-timeout` value.

Reply:
good point. I'm kinda arbitrarily setting it to 2/3's right now, figured with the current 15 minute length that 10 minutes reduces calls but still conservatively is far out from the actual expiration